pyquil/compatibility/v2/__init__.py

```diff
@@ -0,0 +1 @@
+from pyquil.compatibility.v2.api._quantum_computer import QuantumComputer, get_qc
```

Comment:
I think we should omit `QuantumComputer` here and put in the `v2/api/__init__.py` instead, to be consistent with `pyquil.api/__init__.py`.

Overall, it seems to me that the following analogies should hold:
- `compatibility.v2` => `pyquil`
- `compatibility.v2.api` => `pyquil.api`

Reply:
Note: I made the proposed change over in #1344 since it pertained to that work too, so if you're OK with it, let me know and then you don't need to do anything here

Reply:
I agree, good call. Will leave as-is here as suggested